### DISCUSSION TITLE
how to view translations in table ?

### DISCUSSION THREAD

**Discussion by KhaldMansour** [filamentphp/filament#7364] [Help] [answered]
I use laravel spatie for translation and i have a translatable field named (title) I have no problem in saving the field in th database but the problem shows when I want to view this field translated based on the locale switcher variable the user choose.

this is some code snippets from what I tried to solve.
![Screenshot from 2023-08-02 00-57-45](https://github.com/filamentphp/filament/assets/52106756/b78d4f46-1e37-4523-9375-9b1a5f569c4e)
![Screenshot from 2023-08-02 00-57-06](https://github.com/filamentphp/filament/assets/52106756/b0ba3a13-2763-4021-a5b2-0d475df88f87)

**Comment by danharrin** [comment, marked as answer]
Spatie translatable is for translatable content, not the translatable interface. Try this plugin https://filamentphp.com/plugins/bezhansalleh-language-switch
